tests/openvino/utils_tests.py

```diff
@@ -103,6 +103,7 @@
     "opt": "hf-internal-testing/tiny-random-OPTModel",
     "opt125m": "facebook/opt-125m",
     "opt_gptq": "ybelkada/opt-125m-gptq-4bit",
+    "mamba": "rkazants/tiny-mamba",
```

Comment:
would you mind adding a test for falcon mamba as well like for example https://huggingface.co/trl-internal-testing/tiny-FalconMambaForCausalLM